apps/condo/domains/ticket/components/TicketChanges/TicketChange.tsx

```diff
@@ -0,0 +1,178 @@
+                // was changed a whole address of the ticket, so, to preserve context,
+                // the change of the unit is displayed as the change of the address
+                UnitNameChangedMessage
+                    .replace('{address}', ticketChange.ticket.property.address)
+                    .replace('{unitName}', value)
+            ),
+        }
+        return has(formatterFor, field)
+            ? formatterFor[field](field, value)
+            : value
+    }
+
+    const format = (field, value) => (
+        typeof value === 'boolean'
+            ? BooleanToString[field][value]
+            : formatField(field, value)
+    )
+
+    /*
+        Interpolates message string with JSX tags.
+        They will be safely mounted in place of `{to}` and `{from}` placeholders
+     */
+    const formatDiffMessage = (field, message, ticketChange) => {
+        // we have both "from" and "to" parts to interpolate
+        if (message.search('{from}') !== -1 && message.search('{to}') !== -1) {
+            const parts1 = message.split('{from}')
+            const parts2 =  parts1[1].split('{to}')
+            const valueFrom = ticketChange[`${field}From`]
+            const valueTo = ticketChange[`${field}To`]
+            return [
+                <SafeUserMention createdBy={ticketChange.createdBy} key={1}/>,
+                ' ',
+                parts1[0],
+                <del key={3}>{format(field, valueFrom)}</del>,
+                parts2[0],
+                <ins key={2}>{format(field, valueTo)}</ins>,
+                parts2[1],
+            ]
+        } else { // only "to" part
+            const parts =  message.split('{to}')
+            const valueTo = ticketChange[`${field}To`]
+            return [
+                <SafeUserMention createdBy={ticketChange.createdBy} key={1}/>,
+                ' ',
+                parts[0],
+                <ins key={2}>{format(field, valueTo)}</ins>,
+                parts[1],
+            ]
+        }
+    }
+
+    // Omit what was not changed
+    const changedFields = fields.filter(([f]) => (
+        ticketChange[`${f}From`] !== ticketChange[`${f}To`]
+    ))
+
+    return changedFields.map(([field, message]) => ({
+        field,
+        message: formatDiffMessage(field, message, ticketChange),
+    }))
+}
+
+const SafeUserMention = ({ createdBy }) => {
+    const intl = useIntl()
+    const DeletedCreatedAtNoticeTitle = intl.formatMessage({ id: 'pages.condo.ticket.TicketChanges.notice.DeletedCreatedAt.title' })
+    const DeletedCreatedAtNoticeDescription = intl.formatMessage({ id: 'pages.condo.ticket.TicketChanges.notice.DeletedCreatedAt.description' })
+    return (
+        createdBy ? (
+            createdBy.name
+        ) : (
+            <Tooltip placement="top" title={DeletedCreatedAtNoticeDescription}>
+                <span>{DeletedCreatedAtNoticeTitle}</span>
+            </Tooltip>
+        )
+    )
+}
+
+const Diff = styled.p`
+    &.statusDisplayName {
```

Comment:
redurant styles, use antd components.

Reply:
With antd components the code will be unnecessary bloated several times with conditions and different `Typography` components with different props.
Antd components, anyway, needs to be styled with css, — for example, `<Typography.Text delete>` should not have "striketrough" style, but by default it has. I will need to have separate styles for some fields.
It seems to me, that my solution is the shortest one and easier to understand.